File: Problems/1301.-1400./1346._Check_If_N_and_Its_Double_Exist/CheckIfNandItsDoubleExist0.py
#!/bin/python3
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        '''
        Check if there exists any 2 elements in arr, such
        that one is double the other
        '''
        logic = False
        length = len(arr)
        for i in range(0,length-1):
            for j in range(i+1,length):
                if (arr[i] == 2*arr[j] or arr[i]*2 == arr[j]):
                    logic = True
        return logic

File: Problems/1301.-1400./1346._Check_If_N_and_Its_Double_Exist/CheckIfNandItsDoubleExist0.java
//!/bin/java
class Solution {
    public boolean checkIfExist(int[] arr) {
        boolean logic = false;
        for (int i = 0; i < arr.length-1; ++i) {
            for (int j = i+1; j < arr.length; ++j) {
                if (arr[i] == 2*arr[j] || arr[i]*2 == arr[j]) {
                    logic = true;
                    break;
                }
            }
        }
        return logic;
    }
}
